ts/features/zendesk/screens/ZendeskAskPermissions.tsx

```diff
@@ -100,7 +100,7 @@ const getItems = (props: ItemProps): ReadonlyArray<ItemPermissionProps> => [
   },
   {
     id: "profileFiscalCode",
-    icon: <FiscalCodeIcon {...iconProps} />,
+    icon: <CardIcon {...iconProps} />,
```

Comment:
So the icon for FiscalCode and CardIssues is the same?

Reply:
@hevelius yes according to [figma](https://www.figma.com/file/PsDwmBK5bqKZdXsycrTolb/Flusso-Assistenza-(Zendesk)?node-id=12%3A1798&t=3hzGNYnSQd7svFW9-4)